scss/_patterns_navigation.scss

```diff
@@ -161,18 +154,28 @@
             border-bottom: 1px solid $color-mid-light;
           }
         }
-
-        @media (min-width: $breakpoint-navigation-threshold + 1) {
-          padding: $sp-small ($sp-large - $sp-xx-small);
-        }
       }
 
+      .p-navigation__link > a,
```

Comment:
Applying same styles for links both nested under `.p-navigation__link` and directly under `.p-navigation__links`.